rasa/core/policies/keras_policy.py

```diff
@@ -223,23 +218,22 @@ def continue_training(
         batch_size = kwargs.get("batch_size", 5)
         epochs = kwargs.get("epochs", 50)
 
-        with self.graph.as_default(), self.session.as_default():
-            for _ in range(epochs):
-                training_data = self._training_data_for_continue_training(
-                    batch_size, training_trackers, domain
-                )
+        for _ in range(epochs):
+            training_data = self._training_data_for_continue_training(
```

Comment:
does this really have to be in the `for` loop or can we do it once before?

Reply:
see the comment in the code in the beginning of `continue_training` method